ext/mysqli/tests/mysqli_fetch_object_oo.phpt

```diff
@@ -73,15 +73,10 @@ require_once('skipifconnectfailure.inc');
     }
 
     try {
-        $obj = $res->fetch_object('mysqli_fetch_object_construct', null);
-
-        if (($obj->ID !== "3") || ($obj->label !== "c") || ($obj->a !== NULL) || ($obj->b !== NULL) || (get_class($obj) != 'mysqli_fetch_object_construct')) {
-            printf("[009] Object seems wrong. [%d] %s\n", $mysqli->errno, $mysqli->error);
-            var_dump($obj);
-        }
-    } catch (Error $e) {
-        handle_catchable_fatal($e->getCode(), $e->getMessage(), $e->getFile(), $e->getLine());
+        $res->fetch_object('mysqli_fetch_object_construct', null);
+    } catch (TypeError $e) {
```

Comment:
It took me an embarrassingly long time to find this, but ... the problem is that this uses `$e` here and then dumps `$exception`, which points to an earlier exception :(

Reply:
OMG... I was also searching for the problem in the morning, but didn't find it 🤦